src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -17,90 +17,108 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({ users, control,
     <PageBlock title="Project Details">
       <Grid container spacing={1}>
         <Grid item xs={12} md={6} sx={{ mt: 2, mb: 1 }}>
-          <ReactHookTextField
-            name="name"
-            control={control}
-            sx={{ width: 10 / 10 }}
-            label="Project Name"
-            errorMessage={errors.name}
-          />
+          <FormControl sx={{ width: 675 }}>
+            <FormLabel>Project Name</FormLabel>
+            <ReactHookTextField
+              name="name"
+              control={control}
+              placeholder="Enter project name..."
+              errorMessage={errors.name}
+            />
+          </FormControl>
         </Grid>
         <Grid item xs={12} md={6} sx={{ mt: 2, mb: 1 }}>
-          <ReactHookTextField
-            name="budget"
-            control={control}
-            type="number"
-            label="Budget"
-            sx={{ width: 3 / 10 }}
-            icon={<AttachMoneyIcon />}
-            errorMessage={errors.budget}
-          />
+          <FormControl sx={{ width: 200 }}>
+            <FormLabel>Budget</FormLabel>
+            <ReactHookTextField
+              name="budget"
+              control={control}
+              type="number"
+              placeholder="Enter budget..."
+              icon={<AttachMoneyIcon />}
+              errorMessage={errors.budget}
+            />
+          </FormControl>
```

Comment:
<img width="1192" alt="Screen Shot 2023-02-06 at 12 24 43 AM" src="https://user-images.githubusercontent.com/87815804/216890663-7051ca4a-e46f-44cb-b486-ec86cbe3c489.png">


This is what your branch looks like for me. I'm not sure why there's a difference, but maybe try going fullscreen/ getting out of fullscreen and see what it looks like.